Dockerfile

```diff
@@ -140,25 +140,15 @@ RUN apt-get update -qq \
         libxtst6 \
         git \
         pkg-config \
+        wget \
+        bzip2 \
         && rm -rf /var/lib/apt/lists/*
 
+# Instlalling headless browsers
 RUN --mount=type=cache,target=/root/.cache/pip \
     pip install playwright
-RUN playwright install-deps
-RUN playwright install chromium
-
-# Install GeckoDriver WebDriver
-ARG GECKODRIVER_VERSION=v0.34.0 \
-    FIREFOX_VERSION=125.0.3
-
-RUN apt-get update -qq \
-    && apt-get install -yqq --no-install-recommends wget bzip2 \
-    && wget -q https://github.com/mozilla/geckodriver/releases/download/${GECKODRIVER_VERSION}/geckodriver-${GECKODRIVER_VERSION}-linux64.tar.gz -O - | tar xfz - -C /usr/local/bin \
-    # Install Firefox
-    && wget -q https://download-installer.cdn.mozilla.net/pub/firefox/releases/${FIREFOX_VERSION}/linux-x86_64/en-US/firefox-${FIREFOX_VERSION}.tar.bz2 -O - | tar xfj - -C /opt \
-    && ln -s /opt/firefox/firefox /usr/local/bin/firefox \
-    && apt-get autoremove -yqq --purge wget bzip2 && rm -rf /var/[log,tmp]/* /tmp/* /var/lib/apt/lists/*
-# Cache everything for dev purposes...
+RUN playwright install chromium --with-deps
+RUN playwright install firefox --with-deps
```

Comment:
I'm curious, does with work with multiarch builds? I'm assuming the chromium and firefox binaries are different on `x86_64` and `arm64`. It seems previously we were only installing the `x86_64` versions.

Reply:
@mistercrunch According to the [docs](https://playwright.dev/docs/browsers#managing-browser-binaries), playwright installs the browsers in some local directory, not in `usr/local/bin`. We can change that with `PLAYWRIGHT_BROWSERS_PATH` env variable

Reply:
@villebro I think `playwright` is smart and figures out what platform its own and downloads the right bins. 

@kgabryje is targeting `usr/local/bin` a requirement? seems on this build cypress e2e is working as expected (?) Not sure how to more directly test it than just trusting cypress.

Reply:
I think cypress installs its own chromium binary. A proper way to test it would be to run a report